service/src/test/resources/view/elasticview-source.json

```diff
@@ -27,6 +27,18 @@
         "type": "keyword",
         "copy_to": "_all_fields"
       },
+      "prefLabel": {
```

Comment:
Other fields are prefixed by _, why not those ?
I don't know if there is any norm so just asking.
Anf maybe they could be included in an object so that we have:
container  { prefLabel, label, name }